input/samples/vn_simona/fullGrid/wec_type_input.csv

```diff
@@ -1,2 +1,2 @@
-"uuid","capex","cos_phi_rated","cp_characteristic","eta_conv","hub_height","id","opex","rotor_area","s_rated"
-b70374f3-1feb-45af-8011-1304a532d398,0.0,0.9,"cP:{(0.0,0.0),(1.0,0.0),(2.0,0.115933516),(3.0,0.286255595),(4.0,0.39610618),(5.0,0.430345211),(6.0,0.45944023),(7.0,0.479507331),(8.0,0.492113623),(9.0,0.500417188),(10.0,0.488466547),(11.0,0.420415054),(12.0,0.354241299),(13.0,0.288470591),(14.0,0.230965702),(15.0,0.18778367),(16.0,0.154728976),(17.0,0.128998552),(18.0,0.108671106),(19.0,0.09239975),(20.0,0.079221236),(21.0,0.068434282),(22.0,0.059520087),(23.0,0.052089249),(24.0,0.045845623),(25.0,0.040561273),(26.0,0.036058824),(27.0,0.032198846),(28.0,0.016618264),(29.0,0.010330976),(30.0,0.006091519),(31.0,0.003331177),(32.0,0.001641637),(33.0,0.000705423),(34.0,0.000196644),(35.0,0.0),(36.0,0.0),(37.0,0.0),(38.0,0.0),(39.0,0.0),(40.0,0.0),(41.0,0.0),(42.0,0.0),(43.0,0.0),(44.0,0.0),(45.0,0.0),(46.0,0.0),(47.0,0.0),(48.0,0.0),(49.0,0.0),(50.0,0.0)}",100.0,98.0,EnerconE-82,0.0,5281.02,2222.2
+uuid,capex,cos_phi_rated,cp_characteristic,eta_conv,hub_height,id,opex,rotor_area,s_rated
+b70374f3-1feb-45af-8011-1304a532d398,0.0,0.9,"cP:{(0.0000000000,0.0000000000),(1.0000000000,0.0000000000),(2.0000000000,0.1159335160),(3.0000000000,0.2862555950),(4.0000000000,0.3961061800),(5.0000000000,0.4303452110),(6.0000000000,0.4594402300),(7.0000000000,0.4795073310),(8.0000000000,0.4921136230),(9.0000000000,0.5004171880),(10.0000000000,0.4884665470),(11.0000000000,0.4204150540),(12.0000000000,0.3542412990),(13.0000000000,0.2884705910),(14.0000000000,0.2309657020),(15.0000000000,0.1877836700),(16.0000000000,0.1547289760),(17.0000000000,0.1289985520),(18.0000000000,0.1086711060),(19.0000000000,0.0923997500),(20.0000000000,0.0792212360),(21.0000000000,0.0684342820),(22.0000000000,0.0595200870),(23.0000000000,0.0520892490),(24.0000000000,0.0458456230),(25.0000000000,0.0405612730),(26.0000000000,0.0360588240),(27.0000000000,0.0321988460),(28.0000000000,0.0166182640),(29.0000000000,0.0103309760),(30.0000000000,0.0060915190),(31.0000000000,0.0033311770),(32.0000000000,0.0016416370),(33.0000000000,0.0007054230),(34.0000000000,0.0001966440),(35.0000000000,0.0000000000),(36.0000000000,0.0000000000),(37.0000000000,0.0000000000),(38.0000000000,0.0000000000),(39.0000000000,0.0000000000),(40.0000000000,0.0000000000),(41.0000000000,0.0000000000),(42.0000000000,0.0000000000),(43.0000000000,0.0000000000),(44.0000000000,0.0000000000),(45.0000000000,0.0000000000),(46.0000000000,0.0000000000),(47.0000000000,0.0000000000),(48.0000000000,0.0000000000),(49.0000000000,0.0000000000),(50.0000000000,0.0000000000)}",100.0,98.0,EnerconE-82,0.0,5281.02,2222.2
```

Comment:
Do you know why the `0` decimal places are printed out like that?

Reply:
To avoid rounding in other dimensions (e.g. (2.0000000000, **0.1159335160**)), I used 10 decimal places in the transformation (PSDM).

Reply:
We now got rid of the decimal places parameter, which means that only the amount of digits that make sense for a number should be printed: https://github.com/ie3-institute/PowerSystemDataModel/pull/714
Would you mind trying with the changed PSDM?